src/tribler-gui/tribler_gui/debug_window.py

```diff
@@ -15,6 +15,7 @@
 import psutil
 
 import tribler_core.utilities.json_util as json
+from tribler_core.utilities.utilities import has_bep33_support
```

Comment:
You are importing from the `tribler_core` in the `tribler_gui` module directly. Even though we have more core imports in the GUI code, this can have unintended side-effects, if `tribler_core.utilities.utilities` contains other imports.

Would there be an easy way to request this information from the core through the REST API?

Reply:
I checked utilities file which seems pretty independent from core so I did the import. Otherwise, I could simply replace the function call with `'dht_pkt_alert' in dir(libtorrent)`. What do you suggest?

Reply:
On second thoughts, let's not try to overdo this for now. It's the debug panel, and a search through the GUI code base reveals _many_ architectural violations regarding imports from the core 👍